src/patternfly/components/Table/table-grid.scss

```diff
@@ -105,16 +111,40 @@
   // Table toggle icons
   --pf-c-table__toggle__icon--Transition: .2s ease-in 0s;
   --pf-c-table__toggle--m-expanded__icon--Transform: rotate(180deg);
-
-  // End responsive vars
-  // ==================================================================
 }
 
 // Check table cell
 @include pf-mobile-layout {
+  --pf-c-table--cell--PaddingTop: var(--pf-c-table--m-grid--cell--PaddingTop);
+  --pf-c-table--cell--PaddingRight: var(--pf-c-table--m-grid--cell--PaddingRight);
+  --pf-c-table--cell--PaddingBottom: var(--pf-c-table--m-grid--cell--PaddingBottom);
+  --pf-c-table--cell--PaddingLeft: var(--pf-c-table--m-grid--cell--PaddingLeft);
+
   display: grid;
   border: none;
 
+  // reset cell modifications
+  &.pf-c-table tr > * {
+    width: auto;
+    min-width: 0;
+    max-width: none;
+    overflow: visible;
+    text-overflow: clip;
+    white-space: normal;
+  }
+
+  // apply modifications to text
+  .pf-c-table__text {
+    position: relative;
+    display: block;
+    width: auto;
+    min-width: 0;
+    max-width: none;
+    overflow: var(--pf-c-table--cell--Overflow);
```

Comment:
could you use the overflow mixin here ?

Reply:
I think using the overflow mixin wouldn't apply to this use case as I'm using the variables specifically to pass the values from the parent `th/td` to the selector. Looks like the overflow mixin would just hide the overflow, rather than respond to the new modifiers.